ARMTemplates/SnoozeVIS/azuredeploy.json

```diff
@@ -0,0 +1,792 @@
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "LogicAppName": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "The name of the logic app that creates start/stop schedule."
+      }
+    },
+    "SAPOperationType": {
+      "allowedValues": ["start", "stop"],
+      "type": "string",
+      "metadata": {
+        "allowedValues": ["start", "stop"],
+        "description": "Operation to be performed on SAP resource.",
+        "displayName": "SAP operation type",
+        "required": true
+      }
+    },
+    "IncludeDatabaseOperations": {
+      "type": "bool",
+      "metadata": {
+        "description": "Start/Stop the Database Instance",
+        "displayName": "Include Database instance",
+        "required": false
+      }
+    },
+    "IncludeVirtualMachineOperation": {
+      "type": "bool",
+      "metadata": {
+        "description": "Start/Stop the virtual machines",
+        "displayName": "Include virtual machines operation",
+        "required": true
+      }
+    },
+    "OperationExecution": {
+      "type": "string",
+      "metadata": {
+        "description": "Perform operations in sequence",
+        "displayName": "Perform operations in sequence",
+        "required": true
+      }
+    },
+    "SoftStopTimeout": {
+      "type": "int",
+      "defaultValue": 0,
+      "metadata": {
+        "required": false,
+        "displayName": "Soft stop timeout",
+        "description": "Soft stop timeout value, if this value is 0, hard stop will be triggered"
+      }
+    },
+    "VirtualInstanceForSAPSolutionsResourceID": {
+      "type": "array",
+      "metadata": {
+        "displayName": "Virtual Instance for SAP solutions Resource ID(s)",
+        "required": true
+      }
+    },
+    "Frequency": {
+      "allowedValues": ["Month", "Week", "Day"],
+      "type": "string",
+      "metadata": {
+        "allowedValues": ["Month", "Week", "Day"],
+        "description": "How often should schedule run. i.e. Weekly",
+        "displayName": "Frequency",
+        "placeholder": "How often does schedule run",
+        "required": false
+      }
+    },
+    "Interval": {
+      "type": "int"
+    },
+    "OnTheseDays": {
+      "type": "array",
+      "defaultValue": [],
+      "metadata": {
+        "description": "Remove the days in a week you want to skip the opeartion",
+        "required": false,
+        "displayName": "On these days"
+      }
+    },
+    "StartTime": {
+      "type": "string",
+      "metadata": {
+        "description": "Select start time to Power On SAP system",
+        "displayName": "Start Time",
+        "format": "timeFormat",
+        "required": false
+      }
+    },
+    "TimeZone": {
+      "type": "string",
+      "metadata": {
+        "description": "Select time zone for task schedule",
+        "displayName": "Time zone",
+        "placeholder": "Start VM at this specific time zone",

```

Comment:
Should this be same as description?
Select time zone for task schedule

Reply:
Did you mean the placeholder?